remo/events/models.py

```diff
@@ -282,7 +282,7 @@ def email_event_owner_on_add_comment(sender, instance, **kwargs):
     owner = instance.event.owner
     event_url = reverse('events_view_event', kwargs={'slug': event.slug})
     ctx_data = {'event': event, 'owner': owner, 'user': instance.user,
-                'comment': instance.comment, 'event_url': event_url}
+                'comment': instance.comment, 'event_url': event_url, 'commenter': instance.user}
```

Comment:
You could replace the `user` entry completely here with the `commenter` because the user parameter is not used anywhere in the template.

Reply:
Thanks for the response. Then i will change and add another commit to this PR. fine?